tasks/copy_to_glacier_cumulus_translator/schemas/input.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_glacier_cumulus_translator/schemas/input.json",
+	"title": "copy_to_glacier_cumulus_translator Lambda Input",
+	"description": "The input for the copy_to_glacier_cumulus_translator Lambda.",
+	"type": "object",
+	"required": [
+		"granules"
+	],
+	"properties": {
```

Comment:
You will also need to make sure some collection configuration information gets passed in as well from the CMA. Namely, the information related to the excluded file types and the regex configuration for the files I believe that copy_to_glacier uses to determine what files to archive and the key.

Reply:
Looks like that comes from the config, which is defined in a workflow on a per-lambda basis. Am I misunderstanding something?

Reply:
ok, I wasn't sure if that was needed as part of your JSON description or not.